editor.planx.uk/src/@planx/components/FileUploadAndLabel/SelectMultiple.tsx

```diff
@@ -355,7 +355,7 @@ export const SelectMultiple = (props: SelectMultipleProps) => {
           },
           popper: {
             sx: {
-              boxShadow: 10,
+              boxShadow: "0 0 10px 4px rgba(0, 0, 0, 0.5)",
```

Comment:
Moving this out of `theme.ts` as it was being applied across all popper instances and breaking tooltips.

Reply:
Thanks for spotting this one